tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -3297,6 +3297,77 @@ TEST_P(HopperMatmulSchedulerTest, FusedMultiplySum) {
   tref = atMatmul(A.squeeze(), B.squeeze(), layout);
 }
 
+TEST_P(HopperMatmulSchedulerTest, FusedMultiplySumBiasNeg) {
```

Comment:
Please put a TODO comment to remove this test once we have merged #3440  and unguarded other tests that already exercise the same functionality on Ampere.